opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/EpsilonAdaptationDefault.java

```diff
@@ -0,0 +1,22 @@
+package org.opt4j.optimizers.ea.aeseh;
+
+/**
+ * The {@link EpsilonAdaptationDefault} adapts the ε values exactly as described
+ * in the paper cited in the {@link AeSeHModule}.
+ * 
+ * @author Fedor Smirnov
+ *
+ */
+public class EpsilonAdaptationDefault implements EpsilonAdaptation {
```

Comment:
Isn`t this one such specific that the separation of class and interface is superfluous? If you can think about an alternative implementation, what is then specific about this one? -> If you have an answer, use it in the class name instead of "Default" and also in the documentation.